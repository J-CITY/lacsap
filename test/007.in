program HelloWorld;
var mas : array [1..10] of char;
    int : integer;
    r : real;
begin
    int := 1+(2*3)-(((2-4)-1)-2);
    mas[2] := (mas[1]*(1-mas[5]) - (mas[1]-mas[(1+3*2)]);
end.
