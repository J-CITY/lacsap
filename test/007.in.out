End   0   0         keyword    39
END   0   4         keyword    39
end   0   8         keyword    39
